UI/src/app/layout/navigation/navigation.component.ts

```diff
@@ -16,17 +17,22 @@ export class NavigationComponent implements OnInit {
     return this.isAuthenticated;
```

Comment:
maybe it's better to make isAuthenticated field private in the sake of incapsulation 
so you will have this getter public (as it is now) and the field itself will be safe from any unexpected changes